lib/src/nesting_level.dart

```diff
@@ -25,7 +25,7 @@ class NestingLevel extends FastHash {
   /// The nesting level surrounding this one, or `null` if this is represents
   /// top level code in a block.
   NestingLevel? get parent => _parent;
-  NestingLevel? _parent;
+  final NestingLevel? _parent;
```

Comment:
Now that it is `final` you can make the field public and remove the getter.

Reply:
Ah, good catch. Done.